java/client/src/main/java/glide/api/commands/ScriptingAndFunctionsClusterCommands.java

```diff
@@ -267,6 +270,111 @@ CompletableFuture<ClusterValue<Map<String, Object>[]>> functionList(
      */
     CompletableFuture<String> functionDelete(String libName, Route route);
 
+    /**
+     * Returns the serialized payload of all loaded libraries.<br>
+     * The command will be routed to all primary nodes.
+     *
+     * @since Redis 7.0 and above.
+     * @see <a href="https://redis.io/docs/latest/commands/function-dump/">redis.io</a> for details.
+     * @return The serialized payload of all loaded libraries.
+     * @example
+     *     <pre>{@code
+     * ClusterValue<byte[]> data = client.functionDump().get();
+     * // data contains the serialized dump from all primary nodes
+     * // now data could be saved to restore loaded functions on any Redis instance
+     * }</pre>
+     */
+    CompletableFuture<ClusterValue<byte[]>> functionDump();
```

Comment:
As of [#159](https://github.com/amazon-contributing/redis-rs/pull/159) is closed unmerged
```suggestion
     * byte[] data = client.functionDump().get();
     * // now data could be saved to restore loaded functions on any Redis instance
     * }</pre>
     */
    CompletableFuture<byte[]> functionDump();
```

Reply:
That is the reason I didn't add tests yet